scripts/set_apigateway_base_path_mapping.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+"""
+This script idempotently maps the AWS Apigateway custom domain name to the Apigateway
```

Comment:
Nit: Apigateway is a really odd spelling. `Amazon API Gateway` is the official spelling.